oorq/oorq.py

```diff
@@ -93,6 +93,8 @@ def num_jobs(self):
 
     @property
     def progress(self):
+        if not self.num_jobs:
```

Comment:
Crec que s'hauria de posar un `and self.joined` així assegurem que ja s'ha acabat afegir jobs i evitar*race conditions*